src/components/FilterOptions/ReasonFilterOptions/ReasonFilterOptions.tsx

```diff
@@ -27,8 +29,11 @@ const ReasonFilterOptions: React.FC<Props> = ({ reasons, dispatch }: Props) => {
               }}
             ></input>
             <label className="govuk-label govuk-checkboxes__label" htmlFor={`${reason.toLowerCase()}-type`}>
-              {reason}
+              {reason}{" "}
```

Comment:
Why do we need this space?

Reply:
I've removed it, I'm not sure why I've added this...